server/channels/app/notification_email.go

```diff
@@ -109,7 +109,7 @@ func (a *App) sendNotificationEmail(c request.CTX, notification *PostNotificatio
 		}
 	}
 
-	landingURL := a.GetSiteURL() + "/landing#/" + team.Name
+	landingURL := a.GetSiteURL() + "/" + team.Name
```

Comment:
I am unsure if I should keep "/landing#/" in the URL or not. Some of the tests were written without considering this. If we don't want this change, we must add this format to those tests also.

Reply:
I think we actually need that since it's used for opening email notifications in mobile and/or desktop app if I remember right.

I think adding it to the tests makes sense though. It seems like it was a bit of an oversight that it wasn't added to those to begin with, but I guess they didn't need to care about it until now

Reply:
Done